### PR TITLE
Mvp housing

### PR DESCRIPTION
**Pull request by BuildingCityDashboards**

### CODE REVIEW

File: public/data/Housing/constructionsmonthlies.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-date,Dublin,Dun Laoghaire- Rathdown,Fingal,South Dublin
+date,Dublin City,Dun Laoghaire-Rathdown,Fingal,South Dublin
```
Comment (LiamOSullivan) [inline public/data/Housing/constructionsmonthlies.csv:1]: Rename cols

File: public/data/Housing/constructionsmonthlies.csv [hunk 2]
```diff
@@ -58,4 +58,4 @@ Aug-18,null,null,null,null
 Sep-18,null,null,null,null
 Oct-18,null,null,null,null
 Nov-18,null,null,null,null
-Dec-18,null,null,null,null
```
Comment (LiamOSullivan) [inline public/data/Housing/constructionsmonthlies.csv:61]: ok

**Review by LiamOSullivan — COMMENTED** [review]
Review partial

**Review by BuildingCityDashboards — COMMENTED** [review]
OK

**Pull request merged**